dts/bindings/pinctrl/ti,cc13xx-cc26xx-pinctrl.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2022 Vaishnav Achath
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    TI SimpleLink CC13xx / CC26xx pinctrl node.
+
+    Device pin configuration should be placed in the child nodes of this node.
+    Populate the 'pinmux' field with a pair consisting of a pin number and its IO
+    function
```

Comment:
please extend this documentation with some examples, supported properties, etc. See other vendors as an example.

Reply:
added documentation